src/Umbraco.Core/Services/PackagingService.cs

```diff
@@ -1026,7 +1026,7 @@ private void SavePrevaluesFromXml(List<IDataTypeDefinition> dataTypes, IEnumerab
                 if (prevaluesElement == null) continue;
 
                 var dataTypeDefinitionName = dataTypeElement.Attribute("Name").Value;
-                var dataTypeDefinition = dataTypes.First(x => x.Name == dataTypeDefinitionName);
+                var dataTypeDefinition = dataTypes.FirstOrDefault(x => x.Name == dataTypeDefinitionName);

```

Comment:
if there is a possibility that this value can be null then there needs to be a null check now before _dataTypeService.SavePreValues() further down uses it.